### ISSUE TITLE
incorrect installation steps overview in readme

### ISSUE THREAD

**Issue by jai** [argoflow/argoflow#149]
The [readme](/readme.md) currently states:

> Overview of the steps:
> 
> - fork this repo
> - modify the kustomizations for your purpose
> - run `./setup_repo.sh <your_repo_fork_url>`
> - commit and push your changes
> - install ArgoCD
> - run `kubectl apply -f kubeflow.yaml`

`./setup_repo.sh <your_repo_fork_url>` is incorrect. Referencing the [setup script](/setup_repo.sh) as well as the [argoflow-aws readme](https://github.com/argoflow/argoflow-aws/blob/master/README.md) the correct steps should be something like:

> Overview of the steps:
> 
> - fork this repo
> - modify the kustomizations for your purpose
> - create a `setup.conf` file ([example](/examples/setup.conf)) in the root of the repo
> - run `./setup_repo.sh setup.conf`
> - commit and push your changes
> - install ArgoCD
> - run `kubectl apply -f kubeflow.yaml`

**Comment by tidunguyen** [comment]
I also had the same issue with the outdated docs, folder path and commands no longer work. Took quite a bit of work to figure everything out for new users like me. Even after I got everything on argocd, many components are broken.
 Seems like the project is now poorly maintained.

**Comment by tidunguyen** [comment]
Yes, there are quite a few prequisites like sealed-secrets and yq
